.circleci/config.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - run:
           name: use markdownlint v0.4.0 to lint markdown file (https://github.com/markdownlint/markdownlint)
           command: |
-            find  ./ -name  "*.md" | grep -v vendor | grep -v commandline |  grep -v .github |  grep -v swagger |  grep -v api |  xargs mdl -r ~MD010,~MD013,~MD024,~MD029,~MD033,~MD036
+            find  ./ -name  "*.md" | grep -v vendor | grep -v commandline |  grep -v .github |  grep -v swagger |  grep -v api |  xargs mdl -r ~MD009,~MD010,~MD013,~MD022,~MD024,~MD029,~MD031,~MD033,~MD036
```

Comment:
Why to ignore these rule? @Starnop 
I think we could obey to the rules rather than avoid them to make the CI pass.

Reply:
I'm sure I have followed  the rules defined by `markdown lint`, however , it will always fail to  pass the  check. It's so confusing. But now, I tried to change the name of the file, everything is OK.  ^_^